Packs/PhishTank/ReleaseNotes/2_0_3.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### PhishTank v2
+The 'Additional details' url in the warroom entry is now clickable.
```

Comment:
@Noy-Maimon - please provide more detail here - in what command are we talking? Also please use the RN documentation template.

Reply:
@roysagi Thanks, updated now - can you please verify? thanks again